src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+
+      return this.result;
```

Comment:
you can just return `this` (add for other operation as well)
```suggestion
      return this;
```